site/content/help/feature/custom-textures/index.md

```diff
@@ -3,18 +3,54 @@ title = "Custom Textures"
 description = "Custom texture features for displaying custom graphics packs."
 +++
 
-Citra has the ability to dump game textures and load custom texture packs. Currently, only **`.PNG`** files are supported, but it is expected that more formats will come in the future.
+Citra has the ability to dump game textures and load custom texture packs. Game textures will be dumped as **`.png`** files. Texture packs may contain either **`.png`**, **`.dds`** or **`.ktx`** files. Supported compression formats include **`BCn`** and the various **`ASTC`** block variations.
 
 ## Instructions for dumping textures
 
 * Open `Emulation > Configure...` in Citra's menu and go to `Graphics > Enhancements`.
 * Enable **`Dump textures`** and click **`OK`**.
-* Now open a game of your choice, and start playing. As you keep playing, the textures used by the game will be dumped as **`.PNG`** files.
+* Now open a game of your choice, and start playing. As you keep playing, the textures used by the game will be dumped as **`.png`** files.
   - Right-click on your game in the games list and select **`Open Texture Dump Location`** to open the dump folder.
+  - The dump folder will contain a template pack.json file, that is used for configuration options. This file **must** be copied in the load folder.
 
 ## Instructions for replacing textures
 
 * Right-click on your game in the games list and select **`Open Custom Texture Location`** to open the folder where custom textures will be loaded from.
-* Place your custom texture **`.PNG`** files in the folder.
+* Place your custom texture **`.png`** files in the folder.
 * In `Emulation > Configure... > Graphics > Enhancements`, enable **`Use Custom Textures`** and click **`OK`**.
   - Additionally, if you want your custom textures to be pre-loaded to RAM, enable **`Preload Custom Textures`** as well. This will help improve the performance but will also increase memory usage.
+
+## Texture dumping
+
+Citra will dump textures when they are refereced by the game and uploaded from the guest VRAM to the host memory. Dumped textures may only have power-of-two dimensions in order to avoid dumping host framebuffers.
```

Comment:
```suggestion
Citra will dump textures when they are used by the game and uploaded from the guest VRAM to the host memory. Dumped textures may only have power-of-two dimensions in order to avoid dumping host framebuffers.
```